x-pack/qa/password-protected-keystore/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
The reason this is created in x-pack is that calling the reload settings api with a password require transport SSL to be configured. We could move this to `qa` and run with `OSS` too,  if we limit this to an 1 node cluster